lib/models/Account.js

```diff
@@ -29,7 +35,37 @@ export default class Account extends Model {
     this.id = id;
     this.name = opts.name || '';
     this.isPrimary = opts.isPrimary || false;
-    this.isReadOnly = opts.isReadOnly || false;
+    this.capabilities = Utils._nullOrNewInstance(opts.capabilities, AccountCapabilities);
+    this.mail = Utils._nullOrNewInstance(opts.mail, MailCapabilities);
+    this.contacts = Utils._nullOrNewInstance(opts.contacts, Capabilities);
+    this.calendars = Utils._nullOrNewInstance(opts.calendars, Capabilities);
+  }
+
+  /**
+   * Returns whether this `Account` supports mail or not.
+   *
+   * @returns {Boolean} _true_ if and only if this `Account` has mail enabled, _false_ otherwise.
+   */
+  hasMail() {
+    return this.mail !== null;
```

Comment:
I thought about it, but the spec mandates `null` so I'd rather check that... Do you think it's a problem?